Dockerfile

```diff
@@ -20,7 +20,7 @@ COPY glide.lock /go/src/github.com/deis/workflow-cli/
 
 WORKDIR /go/src/github.com/deis/workflow-cli
 
-RUN glide install --strip-vcs --strip-vendor
```

Comment:
Any particular reason you're removing strip vcs? This has some nice space savings since this uses `deis/pkg` which depends on kuberenetes.